hack/test-cmd.sh

```diff
@@ -310,7 +320,7 @@ runTests() {
   kubectl config set clusters.test-cluster.certificate-authority-data "$cert_data" --set-raw-bytes
   r_writen=$(kubectl config view --raw -o jsonpath='{.clusters[?(@.name == "test-cluster")].cluster.certificate-authority-data}')
 
-  encoded=$(echo -n "$cert_data" | base64 --wrap=0)
+  encoded=$(echo -n "$cert_data" | base64)
```

Comment:
it's nonstandard, so it breaks on osx, and it's unnecessary, because the go base64 decoder skips newlines